AndroidTriviaFragment/app/build.gradle

```diff
@@ -52,5 +52,5 @@ dependencies {
     implementation "androidx.appcompat:appcompat:$supportlibVersion"
     implementation 'androidx.constraintlayout:constraintlayout:2.0.0-alpha1'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
-    implementation "com.google.android.material:material:1.0.0-rc01"
+    implementation "com.google.android.material:material:1.3.0-alpha02"
```

Comment:
Can we use 1.2.0 since that's the stable version?

Reply:
This version was suggested by AS stable version. Some libraries we are using alpha anyway.